test/AffInterface.purs

```diff
@@ -55,7 +54,6 @@ suite = do
     test "Get ChainTip" testGetChainTip
     test "Get waitUntilSlot" testWaitUntilSlot
     test "Get EraSummaries" testGetEraSummaries
-    test "Get ProtocolParameters" testGetProtocolParameters
```

Comment:
No point in testing `getProtocolParameterAff, it's necessarily called to setup a contract, and `getProtocolParameter` is just asks, which doesn't need testing in the way we do here